pam/integration-tests/testdata/golden/TestNativeAuthenticate/Authenticate_user_and_accept_password_reset

```diff
@@ -11,6 +11,13 @@ Choose your provider:
   2. ExampleBroker
```

Comment:
> ```
> pam/integration-tests/native: Show visible typed values in golden files
> 
> We were just showing these values at later point, but it's better to
> record more frames, so that we know what's happened in case of failures.
> ```

I only see golden files and tapes updated by this commit, where is the change which caused these to be updated?

Reply:
Oh it's the tapes which you edited, which in turn caused the golden files to be updated